src/libtexture/imagecache_pvt.h

```diff
@@ -916,14 +916,18 @@ class ImageCacheImpl : public ImageCache {
     /// Try to avoid looking to the big cache (and locking) most of the
     /// time for fairly coherent tile access patterns, by using the
     /// per-thread microcache to boost our hit rate over the big cache.
-    /// Inlined for speed.  The tile is marked as 'used'.
-    bool find_tile(const TileID& id, ImageCachePerThreadInfo* thread_info)
+    /// Inlined for speed.  The tile is marked as 'used' if it wasn't the
+    /// very last one used, or if it was the same as the last used and
+    /// mark_same_tile_used is true.
+    bool find_tile(const TileID& id, ImageCachePerThreadInfo* thread_info,
+                   bool mark_same_tile_used = true)
```

Comment:
This seems like something sensitive enough that it might be worth not  defaulting the argument so that we can truly see everywhere that calls it?

Reply:
sure